contracts/DxMgnPool.sol

```diff
@@ -146,6 +160,9 @@ contract DxMgnPool {
     
     function calculateClaimableMgn(Participation memory participation) private returns (uint) {
         uint duration = auctionCount - participation.startAuctionCount;
+        if (totalPoolSharesCummulative == 0) {
```

Comment:
Is this necessary? If there was no totalPoolShareCummulative, this means there was no MGN minted, so it's fine to fail, isn't it?

Reply:
Yeah, it was just good for debugging. I hate reverts without error messages. /0 is one of them